broken link fix in multiple files

ronak-pal1 opened the pull request.

ronak-pal1 commented:
Sorry, I forgot to sign my commits.

ronak-pal1 commented:
Can anyone review it?

Nlea commented on docs/notebooks.md:
```diff
@@ -44,7 +44,7 @@ Everything you write in a notebook is a **cell**. A cell can contain text (in it
 
 Most cells have the ability to display rich text in them. You can **bold**,
-*italicize*, highlight,  ~~strikethrough~~ content or insert `in-line code` and [links](https://www.notion.so/fiberplane/docs.fiberplane.com). Cells include rich markdown support so you can type all of the above characters using their respective syntax:
+*italicize*, highlight,  ~~strikethrough~~ content or insert `in-line code` and [links](https://docs.fiberplane.com). Cells include rich markdown support so you can type all of the above characters using their respective syntax:
```
Thanks for finding this. Could you include monospacing here so it shows the full [links](docs.fiberplane.com) path

ronak-pal1 commented:
> Dear @ronak-pal1 ,
> 
> Thank you for your contribution. Thank you for finding the broken image path and fixing it fixed.
> 
> The changes to the URL are not correct or needed. I added to comments in your PR
> 
> We also exclude the package json. Could you update your PR, excluding the package.json and including the other two comments to the URLs?
> 
> thank you

Thank you for highlighting my mistakes. I have remoted those commits but can you tell me how can I monospace in notebooks.md file as you said to do monospace. I have no idea.

ronak-pal1 commented:
> Dear @ronak-pal1 ,
> 
> Thank you for your contribution. Thank you for finding the broken image path and fixing it fixed.
> 
> The changes to the URL are not correct or needed. I added to comments in your PR
> 
> We also exclude the package json. Could you update your PR, excluding the package.json and including the other two comments to the URLs?
> 
> thank you

Thank you for highlighting my mistakes. I removed those commits and resolved the issue. Please check once.

Nlea commented:
Hey, I am closing this because now the PR is empty without any changes
